src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -96,6 +96,26 @@ contract BLSRegistryCoordinatorWithIndices is Initializable, IBLSRegistryCoordin
         return _operators[operator].operatorId;
     }
 
+    /// @notice Returns the indices of the quorumBitmaps for the provided `operatorIds` at the given `blockNumber`
+    function getQuorumBitmapIndicesByOperatorIdsAtBlockNumber(uint32 blockNumber, bytes32[] memory operatorIds) external view returns (uint32[] memory) {
+        uint32[] memory indices = new uint32[](operatorIds.length);
+        for (uint256 i = 0; i < operatorIds.length; i++) {
+            uint32 length = uint32(_operatorIdToQuorumBitmapHistory[operatorIds[i]].length);
+            for (uint32 j = 0; j < length; j++) {
+                if(_operatorIdToQuorumBitmapHistory[operatorIds[i]][length - j - 1].updateBlockNumber <= blockNumber) {
+                    require(
+                        _operatorIdToQuorumBitmapHistory[operatorIds[i]][length - j - 1].nextUpdateBlockNumber != 0 ||
```

Comment:
possible bug: shouldn't this be `==0` instead of `!=0`?

Reply:
06bca742 good find